module-domain/src/main/kotlin/site/yourevents/guest/domain/Guest.kt

```diff
@@ -0,0 +1,23 @@
+package site.yourevents.guest.domain
+
+import site.yourevents.invitation.domain.Invitation
+import site.yourevents.member.domain.Member
+import java.util.UUID
+
+class Guest(
+    val id: UUID?,
```

Comment:
JPA 특성상 DB에 커밋될 때 식별자가 생기기에, 현재 서비스처럼 도메인 <-> ORM이 분리되어 있으면 식별자를 알 수가 없습니다. 
따라서 이를 해결하기 위해 Member 생성 시에는 별도로 UUID 식별자를 제외한 정보들만 담는 책임을 지니는 MemberVO data class를 별도로 만들어 주었는데요, Guest 도메인도 마찬가지로 GuestVO 클래스를 별도로 만들어주는 것은 어떻게 생각하시나요?
```suggestion
    val id: UUID,
```

Reply:
슬랙에서 말한 것을 고려하여 VO를 이용하여 구현하였고 ?(엘비스 연산자)를 제외하도록 구현하였습니다!!